server/clip_server/model/clip_trt.py

```diff
@@ -34,77 +35,77 @@ def __init__(
         name: str = None,
     ):
         if name in _MODELS:
+            self._name = name
             cache_dir = os.path.expanduser(f'~/.cache/clip/{name.replace("/", "-")}')
-            self._textual_path = _download(_S3_BUCKET + _MODELS[name][0], cache_dir)
-            self._visual_path = _download(_S3_BUCKET + _MODELS[name][1], cache_dir)
+
+            self._textual_path = os.path.join(
+                cache_dir,
+                f'textual.{ONNX_MODELS[name][0][1]}.trt',
+            )
+            self._visual_path = os.path.join(
+                cache_dir,
+                f'visual.{ONNX_MODELS[name][1][1]}.trt',
+            )
```

Comment:
The local onnx model is not validated with MD5?

Reply:
no they are the local trt files and are generated by us
there is no predefined md5 to validate with

Reply:
What I'm asking is actually the existing of onnx model file, not the TRT model. But anyway, I think the following `onnx_model = CLIPOnnxModel(self._name)` involves the downloading and validation procedure. Hence, there is no problem about this part.